PSOperationsTests/PSOperationsTests.swift

```diff
@@ -850,4 +850,49 @@ class PSOperationsTests: XCTestCase {
         waitForExpectationsWithTimeout(0.5, handler: nil)
         
     }
+    
+    func testRetryOperation() {
```

Comment:
Way to write some tests. Thanks. If you want, you could always add tests in a different file too. We probably need to break them out into separate files for each class we're testing.